client/payment-methods/test/index.js

```diff
@@ -200,4 +244,43 @@ describe( 'PaymentMethods', () => {
 
 		expect( disableUPEButton ).not.toBeInTheDocument();
 	} );
+
+	test( 'clicking "Enable in your store" in express payments enable UPE and redirects', () => {
```

Comment:
You could do this to test if the redirect happens:

```diff
diff --git a/client/payment-methods/test/index.js b/client/payment-methods/test/index.js
index 855908f2..e48c7778 100644
--- a/client/payment-methods/test/index.js
+++ b/client/payment-methods/test/index.js
@@ -245,12 +245,10 @@ describe( 'PaymentMethods', () => {
 		expect( disableUPEButton ).not.toBeInTheDocument();
 	} );
 
-	test( 'clicking "Enable in your store" in express payments enable UPE and redirects', () => {
+	test( 'clicking "Enable in your store" in express payments enable UPE and redirects', async () => {
 		Object.defineProperty( window, 'location', {
-			get() {
-				return {
-					href: 'example.com/',
-				};
+			value: {
+				href: 'example.com/',
 			},
 		} );
 
@@ -280,7 +278,8 @@ describe( 'PaymentMethods', () => {
 		expect( enableInYourStoreButton ).toBeInTheDocument();
 
 		expect( setIsUpeEnabledMock ).not.toHaveBeenCalled();
-		user.click( enableInYourStoreButton );
+		await user.click( enableInYourStoreButton );
 		expect( setIsUpeEnabledMock ).toHaveBeenCalledWith( true );
+		expect( window.location.href ).toEqual( 'admin.php?page=wc-admin&task=woocommerce-payments--additional-payment-methods' );
 	} );
 } );
```

Reply:
Thanks! Added this to verify href.